src/NServiceBus.Core/Licensing/LicenseExpiredForm.cs

```diff
@@ -80,19 +84,62 @@ void browseButton_Click(object sender, EventArgs e)
 
         void PurchaseButton_Click(object sender, EventArgs e)
         {
-            Process.Start("http://particular.net/licensing");
+            Process.Start("https://particular.net/licensing");
         }
 
         void getTrialLicenseButton_Click(object sender, EventArgs e)
         {
-            if (CurrentLicense != null && CurrentLicense.IsTrialLicense)
+            var parameterCollection = new NameValueCollection
             {
-                Process.Start("http://particular.net/extend-your-trial-14");
+                {"NugetUser", IsNugetUser().ToString()},
+                {"PlatformInstaller", HasUserInstalledPlatform().ToString()},
+                {"TrialStartDate", TrialStartDateStore.GetTrialStartDate().ToString()}
```

Comment:
This uses culture specific ToString. Underlying store assumes `trialStart.ToString("yyyy-MM-dd", CultureInfo.InvariantCulture)` should we use a similar ToString pattern here cause depending on the culture this code is called ToString may return different results? Because on my machine the output would be:

```
var trialStart = DateTime.UtcNow;
var trialStartDateString = trialStart.ToString("yyyy-MM-dd", CultureInfo.InvariantCulture);

var trialStartDate = DateTimeOffset.ParseExact(trialStartDateString, "yyyy-MM-dd", CultureInfo.InvariantCulture, DateTimeStyles.AssumeUniversal);

Console.WriteLine(trialStartDate.Date.ToString());
```

`20.09.2016 00:00:00`